draft-ietf-quic-invariants.md

```diff
@@ -235,11 +240,11 @@ A connection ID is an opaque field of arbitrary length.
 
 The primary function of a connection ID is to ensure that changes in addressing
 at lower protocol layers (UDP, IP, and below) don't cause packets for a QUIC
-connection to be delivered to the wrong endpoint.  The connection ID is used by
-endpoints and the intermediaries that support them to ensure that each QUIC
-packet can be delivered to the correct instance of an endpoint.  At the
-endpoint, the connection ID is used to identify which QUIC connection the packet
-is intended for.
+connection to be delivered to the wrong transport endpoint.  The connection ID
```

Comment:
What's a transport endpoint?  I don't recall us using that term elsewhere, so endpoint seems sufficient to me.  Or maybe we need another term for this case?

Reply:
This was really trying to avoid having to pull in the definition of "endpoint" from the transport doc.  I see that I failed there.  Happy to take suggestions, like:

```suggestion
connection to be delivered to the wrong destination.  The connection ID
```

Reply:
"destination" sounds like it could be network-layer?

The document uses "QUIC endpoint" elsewhere to refer to an endpoint at the transport layer, so that may perhaps be the term we are looking for?